website/pages/en/setup.js

```diff
@@ -0,0 +1,110 @@
+const React = require("react");
+
+const fs = require("fs");
+const path = require("path");
+const CompLibrary = require("../../core/CompLibrary.js");
+const Container = CompLibrary.Container;
+const MarkdownBlock = CompLibrary.MarkdownBlock;
+const siteConfig = require(process.cwd() + "/siteConfig.js");
+const setupBabelrc = siteConfig.setupBabelrc;
+const toolsMD = siteConfig.toolsMD;
+
+const StepFour = () => {
+  return (
+    <div className="step-hidden step-setup">
+      <h2>
+        <span className="step-no">4</span> Create <code>.babelrc</code> configuration file
+      </h2>
+      <MarkdownBlock>{setupBabelrc}</MarkdownBlock>
```

Comment:
I guess we'd want to translate the setup too? Not sure how this will work with https://docusaurus.io/docs/en/translation.html#enabling-translations-on-pages unless we wrap in translate but it's also markdown

Reply:
Or maybe we should make individual pages that are translatable and leave this as english?